src/wwwroot/services/settings.js

```diff
@@ -161,5 +163,18 @@
       });
     }
 
+    function getCurrentPlanInfo() {
```

Comment:
Mmm, i think that maybe if the user regrets the chosen plan, and come back to myplan selection, in that case that call will be made again. We should make this as fast as possible to the user.(also we have the other call of this page cached)